template/ci_verify.sh

```
@@ -151,19 +151,15 @@ do
-        # This will fail and exit if the ioc.yaml is invalid
-        # Also show the startup script we just generated (and verify it exists)
-        # 'ibek runtime generate2 /config' reads the whole mounted config folder
-        # (ioc.yaml + any vendored/local *.ibek.support.yaml, proto and db) and
-        # places the generated runtime (st.cmd, proto, db) under /epics/runtime.
+        # Run the full startup sequence in test mode: generates all runtime
+        # assets (st.cmd, db, pvi) exactly as in production, but skips
+        # hardware connections and the IOC binary launch.
+        # Requires ioc-template start.sh to support the --test flag.
         $docker run --rm --entrypoint bash \
-            -v ${service}/config:/config:z \
+            -v ${service}/config:/epics/ioc/config:z \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)